package.json

```diff
@@ -9,8 +9,8 @@
 	"scripts": {
 		"cap:init": "git submodule update --init --recursive",
 		"cap:start": "cd ./cap && dfx deploy ic-history-router && mkdir -p ../.temp && dfx canister id ic-history-router > ../.temp/ic-history-router-id",
-		"dip721:healthcheck": "./.scripts/healthcheck.sh",
-		"dip721:deploy": "./.scripts/deploy.sh",
+		"healthcheck": "./.scripts/healthcheck.sh",
+		"deploy": "./.scripts/deploy.sh",
```

Comment:
Given the original convention, let's keep the prefix for the service, as we have for the Cap dependency, please.

```
"dip721:healthcheck": "./.scripts/healthcheck.sh",
"dip721:deploy": "./.scripts/deploy.sh",
```